client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.html

```diff
@@ -1218,6 +1218,44 @@ <h4 data-translate="cpcivm_addedit_advanced_options_description"></h4>
                                                 </span>
                                             </td>
                                         </tr>
+                                        <!-- available but not activated regions -->
+                                        <tr data-ng-repeat="region in VmAddEditCtrl.availableRegions.data"
+                                            style="cursor: not-allowed;"
+                                            data-ng-if="!VmAddEditCtrl.availableRegions.loading && VmAddEditCtrl.availableRegions.data.length > 0">
+                                            <td style="width: 50px;">
+                                                <flat-radio>
```

Comment:
Could a `oui-radio` be used ? https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/radio